src/main/java/lotto/domain/WinningPrize.java

```diff
@@ -0,0 +1,33 @@
+package lotto.domain;
+
+public enum WinningPrize {
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000);
```

Comment:
자바에서는 가독성을 위해 이렇게 높은 숫자는 언더바를 통해 구분할 수 있습니다. 
```suggestion
    SIX(6, 2_000_000_000);
```